cogito/protocol.go

```diff
@@ -268,6 +277,9 @@ func (src *Source) Validate() error {
 	if len(src.ChatNotifyOnStates) == 0 {
 		src.ChatNotifyOnStates = defaultNotifyStates
 	}
+	if len(src.GithubApiEndpoint) == 0 {
```

Comment:
The parameter is a string, so I would expect `if src.GithubApiEndpoint == ""`, and is also what we use at line 264 above...

Reply:
Changed as suggested.